tools/gulp/tasks/docs.ts

```diff
@@ -1,7 +1,8 @@
 import {task, src, dest} from 'gulp';
 import {Dgeni} from 'dgeni';
-import * as path from 'path';
 import {buildConfig} from 'material2-build-tools';
+import * as fs from 'fs';
```

Comment:
As far as I can tell, there is no `readFileSync` and `writeFileSync` in the docs.ts file